assets/js/components/KeyMetrics/ChangeMetricsLink.test.js

```diff
@@ -56,6 +56,10 @@ describe( 'ChangeMetricsLink', () => {
 	it( 'should not render if no key metrics are selected', () => {
 		provideKeyMetrics( registry, { widgetSlugs: [] } );
 
+		registry
+			.dispatch( 'core/user' )
+			.setUserInputSetting( 'purpose', [ 'test-purpose' ] );
+
```

Comment:
Not sure what this is here for, but when I removed it the tests still pass, so I'm going to remove it 🙂